apm/server/alertsman/server/index.js

```diff
@@ -26,7 +26,7 @@ const {
 
 const parsedUrl = parseMongoUrl(MONGO_URL);
 const oplogFilterNs = `${parsedUrl.dbName}.alerts`;
-const oplogConn = new MongoOplog(MONGO_OPLOG_URL, { ns: oplogFilterNs });
+const oplogConn = MongoOplog(MONGO_OPLOG_URL, { ns: oplogFilterNs });
```

Comment:
You had issues here?

Reply:
Yes, I've got
"TypeError: MongoOplog is not a constructor"
and that fixed it.

Reply:
I think this might be related to the mongo version. I saw that you created a yarn file instead of using `meteor npm i`. 
Can you check your npm versions? 
If you install something with `yarn` or `npm i`, it should be the same version like `meteor npm i`.
Check version with `npm -v` and `meteor npm -v`.
Yarn files have no real advantage anymore since we have the `package-lock.json` file generated by new npm version. Would be great if you remove that too :).